lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -102,6 +104,25 @@ class DetailsRenderer {
     element.appendChild(cardsParent);
     return element;
   }
+
+  /**
+   * @param {!DetailsRenderer.CodeDetailsJSON} details
+   * @return {!Element}
+   */
+  _renderCode(details) {
+    const element = this._dom.createElement('div', 'lh-details');
+
+    const pre = this._dom.createElement('pre', 'lh-code');
+    pre.textContent = details.text;
+
+    if (details.header) {
```

Comment:
`header` seems like it's overloading the term since it's per-item here, maybe there's something better to call it (based on the screenshot, at least?). I don't have any good suggestions, though